Scarb.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "access_control"
+version = "0.1.0"
+cairo-version = "2.3.1"
+authors = ["Lindy Labs"]
+description = "Member-based access control component for Cairo"
+readme = "README.md"
+repository = "https://github.com/lindy-labs/cairo-accesscontrol"
+license-file = "LICENSE"
+keywords = ["access control", "authorization", "cairo", "starknet"]
+
+[dependencies]
+starknet = "2.3.1"
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.11.0" }
```

Comment:
`[dev-dependencies]`

Reply:
This needs to be a dependency because of the imports from `snforge_std` in the test suite. The wadray repo was a special case because it did not require any imports from `snforge_std`.

Reply:
That's exactly it. If it's used only in tests, a project using the library shouldn't need if it's being built for "prod" since the tests aren't built then.

Reply:
This could be a Scarb issue, because if `snforge_std` is in `[dev-dependencies]` instead of `[dependencies]`, then the tests would fail to compile.

Reply:
Hmmmm, that's weird. It does look like a Scarb issue. Leave it like this then and if you feel like it, open an issue in Scarb.

Reply:
Oooh, my bad. `dev-dependencies` are only supported from Scarb 2.4.0 onwards.

Reply:
oh I missed that too, didn't realize I was looking at the latest docs :sweat_smile: